Public/Src/FrontEnd/Nuget/NugetResolver.cs

```diff
@@ -80,79 +80,85 @@ public override async Task<bool> InitResolverAsync(IResolverSettings resolverSet
             {
                 var cgManfiestGenerator = new NugetCgManifestGenerator(Context);
                 string generatedCgManifest = cgManfiestGenerator.GenerateCgManifestForPackages(maybePackages.Result);
-                string existingCgManifest = "INVALID";
 
-                if ( !Configuration.FrontEnd.GenerateCgManifestForNugets.IsValid &&
-                      Configuration.FrontEnd.ValidateCgManifestForNugets.IsValid )
+                if ( Configuration.FrontEnd.ValidateCgManifestForNugets.IsValid )

```

Comment:
```suggestion
                if (Configuration.FrontEnd.ValidateCgManifestForNugets.IsValid && 
                   !Configuration.FrontEnd.ValidateCgManifestForNugets.Equals(Configuration.FrontEnd.GenerateCgManifestForNugets))
```

Reply:
Will this also check validity of Configuration.FrontEnd.GenerateCgManifestForNugets ?

Reply:
No but it doesn't need to.  If ValidateCgManifest is valid and GenerateCgManifest is not, `Equals` will return false so this `if` condition will be satisfied